go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/uwblueprint/shoe-project
 go 1.14
 
 require (
+	github.com/aws/aws-sdk-go v1.37.12
```

Comment:
why do we need the aws sdk?

Reply:
`go mod tidy` caused this.

Maybe because of the s3 stuff?